wowup-electron/src/assets/i18n/fr.json

```diff
@@ -3,7 +3,7 @@
     "ABOUT": {
       "CHANGE_LOG_SECTION_LABEL": "Journal des modifications",
       "TITLE": "WowUp.io",
-      "WEBSITE_LINK_LABEL": "Découvrez le site web!"
+      "WEBSITE_LINK_LABEL": "Découvrez le site web !"
```

Comment:
Extra space here

Reply:
It's not an error. In french, you must add a whitespace before composed symbols like `!`, `?`, `;`.